draft-ietf-httpbis-digest-headers.md

```diff
@@ -233,7 +235,7 @@ Integrity preference fields: collective term for `Want-Representation-Digest` an
 
 The `Representation-Digest` HTTP field can be used in requests and responses to
 communicate digests that are calculated using a hashing algorithm applied to
-the entire selected `representation data` (see {{Section 8.1 of SEMANTICS}}).
+the entire selected  *representation data* (see {{Section 8.1 of SEMANTICS}}).
```

Comment:
above the attention is drawn to the term 

selected representation data

I can't recall if there's a reason this one is different by excluding selected.